addons/pos_sale/report/sale_report.py

```diff
@@ -107,6 +107,9 @@ def _query(self, with_clause='', fields=None, groupby='', from_clause=''):
         if not fields:
             fields = {}
         res = super()._query(with_clause, fields, groupby, from_clause)
+        sale_fields = self._select_additional_fields(fields)
+        for key in sale_fields:
+            fields[key] = ', NULL as %s' % (key)
```

Comment:
This is already managed in `_select` above
```python
        for field in fields.keys():
            select_ += ', NULL AS %s' % (field)
```

Reply:
Yes right @Feyensv,
Improved the code.